clang/test/SemaSYCL/wrapped-accessor.cpp

```diff
@@ -52,8 +52,8 @@ int main() {
 // CHECK-NEXT: ImplicitCastExpr {{.*}} '__global int *' lvalue <NoOp>
 // CHECK-NEXT: DeclRefExpr {{.*}} '__global int *' lvalue ParmVar {{.*}} '_arg_accessor' '__global int *'
 // CHECK-NEXT: ImplicitCastExpr {{.*}} 'range<1>':'cl::sycl::range<1>' <NoOp>
-// CHECK-NEXT: DeclRefExpr {{.*}} 'range<1>':'cl::sycl::range<1>' lvalue ParmVar {{.*}} '_arg_AccessRange' 'range<1>':'cl::sycl::range<1>'
+// CHECK-NEXT: DeclRefExpr {{.*}} 'range<1>':'cl::sycl::range<1>' lvalue ParmVar {{.*}} '{{[a-zA-Z_]+}}' 'range<1>':'cl::sycl::range<1>'
```

Comment:
I think it's better to define variables for kernel parameters and validate that correct parameter is used, if the intention is to omit "exact name" check.

Reply:
Done.